tests/nosetests/pyanaconda_tests/ks_version_test.py

```diff
@@ -27,6 +27,7 @@ class CommandVersionTestCase(unittest.TestCase):
 
     # Names of the kickstart commands and data that should be temporarily ignored.
     IGNORED_NAMES = {
+        "btrfs"
```

Comment:
You should just update the version of the command btrfs to `RHEL8_BTRFS` in `pyanaconda/core/kickstart/commands.py`.